src/ExposureHistory/NextSteps.tsx

```diff
@@ -34,8 +35,21 @@ const CloseButton = () => {
 
 const NextSteps = (): JSX.Element => {
   const navigation = useNavigation()
+  const [connectivity, setConnectivity] = useState<boolean | null | undefined>(
+    null,
+  )
   const { t } = useTranslation()
   useStatusBarEffect("dark-content")
+
+  useEffect(() => {
+    const unsubscribe = NetInfo.addEventListener((state) => {
+      setConnectivity(state.isInternetReachable)
+    })
+    return () => {
```

Comment:
Do you need this return or can you just have `unsubscribe()`?

Reply:
When we return a function in `useEffect` React will run it when component unmounts, so we are using this to clean up this listener on unmount.

Reply:
Oh interesting, didn't know that. Thanks for the explanation!